spec/lib/lighthouse/letters_generator/service_spec.rb

```diff
@@ -13,9 +13,34 @@
     allow_any_instance_of(Lighthouse::LettersGenerator::Configuration).to receive(:connection).and_return(conn)
   end
 
+  context 'a request' do
+    it 'always gets the lighthouse token via get_access_token' do
+      expect_any_instance_of(Lighthouse::LettersGenerator::Configuration)
```

Comment:
Would it make sense to put this in the `before` block since you are repeating this for every test?

Reply:
Great reasoning, came here to make the same comment but I see your point of view.